docs/data-science-get-started.md

```diff
@@ -0,0 +1,37 @@
+# Engage as a Data Scientist
+
+If you are a Data Scientist and would like to engage with this project, here are a few ways to get started and we compile a list of ways you can engage with this project.
+
+## Leverage Open Data Sources
+
+One of the goals of the AI4CI project is to build an open AIOps community centered around open datasets. As a part of this project, we interact with various open source CI data sources and make them available for others to leverage. A detailed overview of our efforts with the various data sources we are working with, along with data collection scripts, and exploratory analyses is available [here](content.md#data-engineering-metrics-and-kpis-for-ci).
+
+## Try out the Notebooks
+
+There are interactive and reproducible notebooks for this entire [project](https://github.com/aicoe-aiops/ocp-ci-analysis) available for anyone to start using on the public [JupyterHub](https://jupyterhub-opf-jupyterhub.apps.zero.massopen.cloud/hub/login) instance on the [Massachusetts Open Cloud](https://massopen.cloud/) (MOC) right now!
+
+1. To get started, access [JupyterHub](https://jupyterhub-opf-jupyterhub.apps.zero.massopen.cloud/), select log in with `moc-sso` and sign in using your Google Account.
```

Comment:
this link will need to be updated with smaug

Reply:
Yes, correct! :) we will also need to update model endpoints, JH URL, Kubeflow pipeline url etc